gammapy/scripts/image_pipe.py

```diff
@@ -21,10 +26,25 @@ def image_pipe_main(args=None):
 
 class ImageAnalysis(object):
     """Gammapy 2D image based analysis.
+
+    center : ``astropy.coordinates import SkyCoord`
+        Center of the image
+    energy_band : `~gammapy.utils.energy.Energy
+        Energy band for which we want to compute the image
+    data_store : `~gammapy.data.DataStore`
+        `DataStore` where are situated the events
     """
 
-    def __init__(self, config):
-        self.config = config
+    def __init__(self, center, energy_band, data_store):
+        self.data_store = data_store
+        self.center = center
+        self.energy_band = energy_band
+        self.counts_image = SkyMap.empty(nxpix=1000, nypix=1000, binsz=0.01, xref=self.center.l.deg,
```

Comment:
Just start to implement the class but I have to leave now so I will continue tomorrow...
not sure If we have to define two members counts_image and bkg_image or if this is better for the method make_counts and make_bkg to return a skymap?
Then for the total_counts_map and total_bkg_map we will define two methods that will iterate on the obs_ids of the datastore,. Should I declare them as members of the class or is it better to have two methods make_total_count and make_total_bkg that will return the total sky map?